src/scss/main.scss

```diff
@@ -10,7 +10,7 @@
 // Layout
 @import 'layout/grid';
 
-//components
+// Components
 @import 'components/hero';
 @import 'components/carousel';
-@import 'components/services';
+@import 'components/header';
```

Comment:
Please make sure to keep both changes here. You are replacing changes from other person.